admin_pages/messages/Messages_Admin_Page.core.php

```diff
@@ -16,23 +16,40 @@
  */
 class Messages_Admin_Page extends EE_Admin_Page
 {
+    /**
+     * @var EEM_Message
+     */
+    private $MSG_MODEL;
+
+    /**
+     * @var EEM_Message_Template
+     */
+    private $MTP_MODEL;
+
+    /**
+     * @var EEM_Message_Template_Group
+     */
+    private $MTG_MODEL;
```

Comment:
added properties and getters for these models because they were called repeatedly with varying names